src/ol/interaction/Link.js

```diff
@@ -52,9 +52,13 @@ function differentArray(a, b) {
   return differentNumber(a[0], b[0]) || differentNumber(a[1], b[1]);
 }
 
+/** @typedef {'x'|'y'|'z'|'r'|'l'} TrackProperties */
+
 /**
  * @typedef {Object} Options
  * @property {boolean|import('../View.js').AnimationOptions} [animate=true] Animate view transitions.
+ * @property {Array<TrackProperties>} [trackProperties=['x', 'y', 'z', 'r', 'l']] Properties to track. Default is to track
```

Comment:
Minor nit, when reading this below, I keep thinking of "track" the noun, like a racetrack or a GPS track.  Feels like `properties` or `params` (maybe overused, but I think people know "query params") would be more natural.